src/components/SettingsPopoup/SettingsPopup.js

```diff
@@ -365,3 +383,16 @@ export default class SettingsPopup extends React.Component {
     );
   }
 }
+
+const mapStateProps = state => ({});
+
+const mapDispatchToProps = {
+  clearCustomColors,
+};
+
+export default compose(
+  connect(
+    mapStateProps,
+    mapDispatchToProps
+  )
```

Comment:
Similar query to the `AudioTransportBar`, the components should have their data passed down consistently

Reply:
I believe we fixed this in newer commits.